src/Aquifer.Jobs/appsettings.json

```diff
@@ -29,6 +29,7 @@
     },
     "OpenAiTranslation": {
         "DefaultLanguageSpecificPromptAppendixFormatString": "Then translate to {0}.",
+        "HtmlAquiferizationBasePrompt": "You receive HTML and then return the HTML with an altered version of the text therein. Never change the formatting of the HTML elements or attributes. For example, if text has an <em> tag around it or a <span> tag with attributes, that tag and its attributes should stay with the text even if it is simplified or moved within the text. Keep the HTML tags and attributes no matter what. Never add a missing html element (i.e. if something starts with <p> but the end is missing </p>, don't add it). Never remove '&nbsp;'.\n\nImprove the clarity of the provided text for a global English audience. Follow these guidelines:\n- Simplify complex sentences for easier understanding.\n- Replace difficult words or technical jargon with simpler alternatives.\n- Add parenthetical explanations when needed.\n- Keep sentences under 25 words.\n- Break long paragraphs into shorter ones.\n- Avoid idioms or figurative language; use straightforward expressions.\n- Use the active voice whenever natural.\n- Add subheadings to improve readability for longer texts.\n- Spell out abbreviations, except BC and AD.\n- Aim for an 8th-9th grade reading level.\n- Preserve the original meaning and intent.\n- Keep the word count close to the original.",
```

Comment:
Here's what this looks like with the newlines formatted. @jwinston-bn what do you think of this?
```
You receive HTML and then return the HTML with an altered version of the text therein. Never change the formatting of the HTML elements or attributes. For example, if text has an <em> tag around it or a <span> tag with attributes, that tag and its attributes should stay with the text even if it is simplified or moved within the text. Keep the HTML tags and attributes no matter what. Never add a missing html element (i.e. if something starts with <p> but the end is missing </p>, don't add it). Never remove '&nbsp;'.

Improve the clarity of the provided text for a global English audience. Follow these guidelines:
- Simplify complex sentences for easier understanding.
- Replace difficult words or technical jargon with simpler alternatives.
- Add parenthetical explanations when needed.
- Keep sentences under 25 words.
- Break long paragraphs into shorter ones.
- Avoid idioms or figurative language; use straightforward expressions.
- Use the active voice whenever natural.
- Add subheadings to improve readability for longer texts.
- Spell out abbreviations, except BC and AD.
- Aim for an 8th-9th grade reading level.
- Preserve the original meaning and intent.
- Keep the word count close to the original.
```

Reply:
Did you test out a long paragraph to see what would happen with the html/tiptap if it broke it up into multiple paragraphs? I'm wondering if we need to do anything to translate a new line to wrap in `<p>` tags or something.

Reply:
Good call, I'll test that